org-super-agenda.el

```diff
@@ -881,16 +881,29 @@ of the arguments to the function."
   "Return function for SELECTOR, or nil if special selector.
 Raise error if invalid selector."
   (cond
-   ((cl-member selector org-super-agenda-special-selectors)
-    ;; Special selector, so no associated function; return nil
-    nil)
-   ;; Valid selector: return function
-   ((plist-get org-super-agenda-group-types selector))
-   ((eq selector :habit)
-    ;; :habit selector used but `org-habit' not loaded
-    (user-error "Please `require' the `org-habit' library to use the :habit selector"))
-   ;; Invalid selector: raise error
-   ((user-error "Invalid org-super-agenda-groups selector: %s" selector))))
+    ((cl-member selector org-super-agenda-special-selectors)
+     ;; Special selector, so no associated function; return nil
+     nil)
+    ;; Valid selector: return function
+    ((plist-get org-super-agenda-group-types selector))
+    ((eq selector :habit)
+     ;; :habit selector used but `org-habit' not loaded
+     (user-error "Please `require' the `org-habit' library to use the :habit selector"))
+    ;; Deprecated selector: raise warning
+    ((when-let ((new-selector (alist-get selector
```

Comment:
I’m using `alist-get`, but I think this is a recent addition to Emacs.  We can use `(cdr (assoc …))` instead.

Reply:
`alist-get` is fine, and I much prefer it over `(cdr (assoc`.  :)

Reply:
Perfect!  I was in the process of rebasing the commits on top of `master`.  Would you like me to still do it?

Reply:
I ended up merging master.

Reply:
Please rebase it.  PRs should generally, if not always, be rebased.

Reply:
Sure, I'll do that instead.